enterprise/audit/table.go

```diff
@@ -4,8 +4,26 @@ import (
 	"reflect"
 
 	"github.com/coder/coder/coderd/database"
+	"github.com/coder/coder/codersdk"
 )
 
+// This mapping creates a relationship between an Auditable Resource
+// and the Audit Actions we track for that resource.
+// It is important to maintain this mapping when adding a new Auditable Resource to the
+// AuditableResources map (below) as our documentation - generated in scripts/auditdocgen/main.go -
+// depends upon it.
+var AuditActionMap = map[string][]string{
```

Comment:
Do you think we can just keep it as `map[string][]Action{}`?

Reply:
@mtojek The reason I didn't do that is because then, in the script, we must range over each Action and convert it to a string, which seemed like extra computation, and since this data structure is only used for that purpose, I thought it might be better to just have it in the format that is easiest for lookup. Let me know if you think this is a suboptimal approach - I'm happy to change it.

Reply:
There are only a few items on this array, so I'm pretty sure that we won't notice the performance drop :) I think that's safe to adjust it.